.github/workflows/kitchen.yml

```
@@ -16,15 +16,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v2
-    - name: 'Install Chef/Ohai from Omnitruck'
-      id: install_chef
-      run: |
-        . { Invoke-WebRequest -useb https://omnitruck.chef.io/install.ps1 } | Invoke-Expression; Install-Project -project chef -channel current
-        $env:PATH = "C:\opscode\chef\bin;C:\opscode\chef\embedded\bin;" + $env:PATH
-        chef-client -v
-        ohai -v
-        rake --version
-        bundle -v
+    - name: Install Infra Client from current channel
+      uses: actionshub/chef-install@master
```

```suggestion
      uses: actionshub/chef-install@main
```

actionshub has switched to the main branch now and @Xorima is looking to remove the master branch shortly 👍🏼